server/plugins/task-plugin-ob-mysql/src/main/java/com/oceanbase/odc/plugin/task/obmysql/datatransfer/factory/LoadParameterFactory.java

```diff
@@ -208,7 +210,10 @@ private void setWhiteListForExternalCsv(LoadParameter parameter, DataTransferCon
         if (!csvFile.exists()) {
             throw new IllegalStateException("Input csv file does not exist");
         }
-        parameter.setInputFile(csvFile);
+        Path path = new Path(csvFile.getAbsolutePath());
+        parameter.setInputFile(parameter.getStorageConfig().getFileSystem().getFileStatus(path));
```

Comment:
Get filestatus return a file?
Here the Logic seems hard to understand

Reply:
Recent versions of ob-loader-dumper switched to hadoop file system API to support both local file system and cloud storage,  FileStatus represents the path of a file.